robot/rover/README.md

```diff
@@ -12,6 +15,14 @@ This method installs it in your arduino libraries folder.
 
 If you wish to refer the `include` statements to set your relative path to the libraries, you must surround the libary name with double quotes.
 
+## StreamDispatcher.py
+Currently this starts a stream via `start_stream.sh` which uses mjpgstreamer, more details are documented in the bash script itself.
+To start the stream, assuming you already have a usb web camera connected, run `./StreamDispatcher.py`. Use `Ctrl + C` to stop the stream.
+
+Assuming the stream is being dispatched, it can be accessed in any browser by visiting: `server_ip:8090/?action=stream`, where `server_ip` is replaced with the actual ip address.
```

Comment:
here you mean the ip address of the machine which is streaming the webcam, right?

Reply:
yes, that would be the server.

Reply:
```suggestion
Assuming the stream is being dispatched, it can be accessed in any browser by visiting: `server_ip:8090/?action=stream`, where `server_ip` is replaced with the actual ip address of the server doing the streaming.
```

Reply:
It doesn't have to be the rover. You can test all this with only one machine serving as both client and server, not necessarily the rover.  So I would add "of the server..." but remove the (the rover in this case). Fair?

Reply:
Done, thanks